### PR TITLE
reformat ivy.inv

### PR DESCRIPTION
**Pull request by lunava**

### CODE REVIEW

**Comment by lunava** [comment]
add_reformatting_checklist_lin_alg

**Comment by lunava** [comment]
add_reformatting_checklist_linear_algebra

**Comment by lunava** [comment]
add_reformatting_checklist_linear_algebra

**Comment by lunava** [comment]
add_reformatting_checklist_linear_algebra

File: ivy/functional/ivy/linear_algebra.py [hunk 1]
```diff
@@ -628,19 +628,60 @@ def inv(
...
+    latest/extensions/generated/signatures.linalg.inv.html>`
+    in the standard.
+
+    Both the description and the type hints above assumes an array input for simplicity,
+    but this function is *nestable*, and therefore also accepts :code:`ivy.Container`
+    instances in place of any of the arguments.
+
     Examples
     --------
+
```
Comment (Daniel4078) [inline ivy/functional/ivy/linear_algebra.py:643]: you have an error here:
ivy/functional/ivy/linear_algebra.py:621 in public function `inv`:
D412: No blank lines allowed between a section header and its content ('Examples')

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Hello, I have a look at the CI tests and currently found one significant error for you to fix (details in the comment below). Now lets wait for the automatic check-list to be added and then we can see if there is anything else you need to do. Thank you!

**Comment by lunava** [comment]
add_reformatting_checklist_linear_algebra

**Comment by lunava** [comment]
add_reformatting_checklist_linear_algebra

**Comment by lunava** [comment]
add_reformatting_checklist_lin_alg

**Comment by lunava** [comment]
add_reformatting_checklist_linearalgebra

**Comment by lunava** [comment]
add_reformatting_checklist_linear_algebra

**Comment by Daniel4078** [comment]
Now with the checklist up and ready, you can edit the status accordingly. the docstring test does not pass currently and I hope the checklist can help you figure out why.

**Comment by lunava** [comment]
Hi, @Daniel4078 can you review my commits? I added the docstring for the inv function.

**Review by Daniel4078 — APPROVED** [review]
It seems like there is nothing wrong with your changes. Good work.

**Pull request merged**